mangaki/mangaki/static/css/skin.css

```diff
@@ -151,80 +151,101 @@ color: #b2B2B2;
 	-o-filter: grayscale(0%);
 }
 
-.mangas-list .manga-sheet .row {
-    margin-left: 0px;
-    margin-right: -15px;
+/****************
+ *  WORK CARDS  *
+ ****************/
+
+.work-sheet {
+    position: relative;
+    margin-bottom: 20px;
+    margin-right: 10%;
+    max-width: 225px;
 }
-.mangas-list .manga-sheet .row > div {
-    padding: 0;
+
+.works-list .work-sheet {
+    margin-left: auto;
+    margin-right: auto;
 }
-.mangas-list .manga-sheet:hover .thumbnail {
-    background-color: #FFF;
-   /*margin-left: -10px;
-    padding-left: 10px;
-    margin-right: -10px;
-    padding-right: 10px;*/
+
+.work-snapshot {
+    width: 100%;
+    display: block;
 }
-/*.mangas-list .manga-sheet:hover .manga-snapshot .manga-snapshot-image {
+
+.work-snapshot-image {
     width: 100%;
-}*/
-.mangas-list .manga-sheet .thumbnail:not(:hover) {
-    /*background-color: #DDD;*/
-    /*border-color: #DDD;*/
+    position: relative;
     opacity: 0.8;
 }
 
-.mangas-list .manga-sheet .manga-snapshot {
-    z-index: 64;
+.work-sheet:hover .work-snapshot-image {
+    opacity: 1;
 }
-.mangas-list .manga-sheet .manga-snapshot .manga-snapshot-image {
-    height: 321px;
-    width: 225px;
-    max-width: 99%;
-    background-size: cover;
-    background-repeat: no-repeat!important;
-
 
+.work-snapshot-image:before {
+    content: "";
+    display: block;
+    padding-top: 142.667%;
 }
 
-.mangas-list .manga-sheet .row .btn {
-    margin-top: 1px;
-    position: relative;
-}
-.mangas-list .manga-sheet .row .btn:hover {
-    z-index: 128;
-}
-.thumbnail {
-  display: block;
-  padding: 4px;
-  margin-bottom: 20px;
-  line-height: 1.42857143;
-  background-color: #fff;
-  border: 0px solid #ddd;
-    border-radius: 0px;
-  -webkit-transition: all .0s ease-in-out;
-  transition: all .0s ease-in-out;
-}
-@media(min-width: 768px) and (max-width: 1000px) {
-    .mangas-list .manga-sheet .row .btn {
-    border-top-left-radius: 0px;
-    border-bottom-left-radius: 0px;
-    }
+.work-snapshot-image img {
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
 }
 
-
-
-.thumbnail .col-md-12 {
+.work-snapshot-title {
+    width: 100%;
     height: 4em;
     line-height: 4em;
+    border-radius: 0 0 10px 10px;
+    box-shadow: #e3e4e6 0px 2px 2px;
 }
 
-.manga-sheet h1 {
-    margin: 0;
+.work-snapshot-title h4 {
     vertical-align: middle;
     display: inline-block;
-    font-size: 1.3em;
-    /* height: 2em; */
+}
+
+.work-votes {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 20%;
+    margin-right: -20%;
```

Comment:
Explique pourquoi il est nécessaire d'utiliser une margin négative (un peu de ["littérature"](http://www.bennadel.com/blog/1174-negative-css-margins-are-not-cool.htm) est contre cette approche, d'où leur gestion délicate dans la codebase).